### ISSUE TITLE
Does JAXv2 allow for arbitrary quantization?

### ISSUE THREAD

**Issue by fabrizio-ottati** [google/aqt#312]
Hi everyone! I would like to use AQT to quantize deep learning models that then I would infer on my hardware (FPGAs). Does JAXv2 support arbitrary quantization (e.g., INT4)? I am asking because I only saw examples using INT8 data type.

**Comment by lukaszlew** [comment]
Yeah, AQT does quantize to INT4, you can set number of bits here:
https://github.com/google/aqt/blob/c1c51bfccd9a9d98a86cce7f716d827e47017d87/aqt/jax/v2/examples/mnist.py#L244
Or you can dive deeper into config system to see how it works if you are quantization expert.

Perhaps we should port our binarized model PokeBNN (https://arxiv.org/abs/2112.00133) to AQTv2... but that requires some work :)

Also unfortunately we don't have a good workflow (yet!) to convert the model to FPGA.
